dom.css

```diff
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  width: 100%;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
+/* style header section */
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  font-size: 17px;
+  border: 1px solid black;
+  text-align: end;
+  width: 100%;
+  height: 50px;
+  background: #f1f1f1;
+  padding: 10px;
+}
+.header a {
+  font-size: 20px;
+  font-weight: 600;
+}
+
+/* Style the submit button */
+form.example button {
+  float: left;
+  width: 20%;
+  padding: 10px;
```

Comment:
Please use correct class names related to element so here we can use form.submit button instead of example and then the comment will also not be needed.